base/dates/periods.jl

```diff
@@ -321,22 +316,14 @@ Base.show(io::IO,x::CompoundPeriod) = print(io,string(x))
 GeneralPeriod = Union{Period,CompoundPeriod}
 (+)(x::GeneralPeriod) = x
 (+){P<:GeneralPeriod}(x::StridedArray{P}) = x
+(-){P<:GeneralPeriod}(x::StridedArray{P}) = broadcast(-, x)
```

Comment:
You're right, it's not needed (there is a general `-(::AbstractArray) method in arraymath.jl); it was left over from some other edits, I think.  I'll delete it.